OOP_Homework_2/BankCount.cs

```diff
@@ -102,6 +102,60 @@ public void PrintCount()
             Console.WriteLine($" Ваш тип счета: {typeOfCount}\n Ваш номер счета: {accountNumber}\n Ваш баланс: {balance}\n ");
 
         }
+
+        public override bool Equals(object obj)
+        {
+            BankCount acc;
+            try
+            {
+                acc = (BankCount)obj;
+            }
+            catch
+            {
+                return false;
+            }
+            return this.accountNumber == acc.accountNumber;
```

Comment:
Не надо так! Это ооооочееееньььь меееедлееееннннооо..... Генерация исключения требует от системы ряда специальных операций. Если есть возможность этого избежать - это НАДО сделать!
```suggestion
            if(obj is not BankCount acc) return false;
            return this.accountNumber == acc.accountNumber;
```